### PR TITLE
fix: Prevent double loading of organization details endpoint

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationContext.jsx [hunk 1]
```diff
@@ -71,9 +71,13 @@ const OrganizationContext = createReactClass({
       this.props.params.orgId &&
       prevProps.params.orgId !== this.props.params.orgId;
 
+    const organizationsLoading =
+      prevProps.organizationsLoading !== this.props.organizationsLoading &&
+      this.props.organizationsLoading === true;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationContext.jsx]: Can you leave a comment why we're waiting on this edge when remounting?

**Review by billyvg — APPROVED** [review]

**Review by markstory — APPROVED** [review]

**Comment by billyvg** [comment]
Do tests pass if we make `OrganizationContext` wait for `this.props.organizationsLoading` to be `false`?

**Comment by lynnagara** [comment]
Since this is an acceptance test, I don't think we can check or wait for props values like that directly

**Comment by billyvg** [comment]
I mean in `componentDidUpdate`, we remount when organizationsLoading is still true.

**Comment by billyvg** [comment]
In account settings there's no org in context, and in that case we go by the last accessed org (via session state), but I'm assuming we don't have that in acceptance tests, so it just chooses the first one from the org list.

**Comment by lynnagara** [comment]
Thanks @billyvg, false makes more sense since we only want to retrigger when it's not loading

**Pull request merged**